packages/loader/driver-utils/src/network.ts

```diff
@@ -45,8 +43,7 @@ export type DriverErrorTelemetryProps = ITelemetryProperties & {
  * Generic network error class.
  */
 export class GenericNetworkError extends LoggingError implements IDriverErrorBase, IFluidErrorBase {
-	// eslint-disable-next-line import/no-deprecated
-	readonly errorType = DriverErrorType.genericNetworkError;
+	readonly errorType = DriverErrorTypes.genericNetworkError;
```

Comment:
```suggestion
	/**
	 * {@inheritDoc IFluidErrorBase.errorType}
	 */
	readonly errorType = DriverErrorTypes.genericNetworkError;
```

(Same with the others)